index.html

```diff
@@ -73,22 +73,22 @@ <h4>MORE FROM YOUTUBE</h4>
     </div>
   </aside>
   <div class="categories">
-    <section class="category-section">
-      <button class="category active">All</button>
-      <button class="category">Category 1</button>
-      <button class="category">Category 2</button>
-      <button class="category">Category 3</button>
-      <button class="category">Category 4</button>
-      <button class="category">Category 5</button>
-      <button class="category">Category 6</button>
-      <button class="category">Category 7</button>
-      <button class="category">Category 8</button>
-      <button class="category">Category 9</button>
-    </section>
+    <div class="category-section">
+      <button class="category active" type="submit">All</button>
+      <button class="category" type="submit">Category 1</button>
+      <button class="category" type="submit">Category 2</button>
+      <button class="category" type="submit">Category 3</button>
+      <button class="category" type="submit">Category 4</button>
+      <button class="category" type="submit">Category 5</button>
+      <button class="category" type="submit">Category 6</button>
+      <button class="category" type="submit">Category 7</button>
+      <button class="category" type="submit">Category 8</button>
+      <button class="category" type="submit">Category 9</button>
+    </div>
   </div>
   <div class="videos">
-    <section  class="video-section">
-      <article class="video-container">
+    <div  class="video-section">
```

Comment:
Consider using fewer containers in this section. Try to structure the elements in a grid.